src/haptics.cpp

```diff
@@ -12,6 +12,41 @@ init_haptics() {
   }
 }
 
+/* Vibrates specified motors for given frequency.
+ *
+ * @param motor_pin	    specified motor to vibrate
```

Comment:
is motor_pin the hardware pin # or the 0-12 index? If it is the 0-12 index, I would change to motor_idx